packages/react-diagrams-core/src/entities/node-layer/NodeLayerModel.ts

```diff
@@ -1,6 +1,7 @@
 import { LayerModel, LayerModelGenerics } from '@projectstorm/react-canvas-core';
 import { NodeModel } from '../node/NodeModel';
 import { DiagramEngine } from '../../DiagramEngine';
+import {DiagramModel} from "../../..";
```

Comment:
nit: code needs to be formatted

Reply:
please also do an explicit import, can sometimes run into circular deps because of this